### PR TITLE
Feat: toast, loaders, 404, emptypage

### PR DESCRIPTION
**Pull request by yashpatel1797**
Feature added.
1. Toast
2. Loaders
3. 404 page not found
4. emptypage

### CODE REVIEW

File: src/components/EmptyData/EmptyData.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import noVideo from "assest/noVideo.png";
+import { Link } from 'react-router-dom';
+import styles from "./EmptyData.module.css"
+const EmptyData = () => {
+    return (
+        <>
+            <div className='text-center'>
```
Comment (ravikumar0403) [inline src/components/EmptyData/EmptyData.js:8]: As you are returning only a div here, fragment is not required

File: src/components/Error/Error.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import error from "assest/error.webp";
+import { Link } from 'react-router-dom';
+const Error = () => {
+    return (
+        <>
+            <div className='text-center'>
```
Comment (ravikumar0403) [inline src/components/Error/Error.js:7]: here as well. fragment is not required

File: src/pages/WatchLater/WatchLater.js [hunk 3]
```diff
@@ -12,17 +12,18 @@ const WatchLater = () => {
...
-                            />
-                        </div>
-                    </div>
+                    {watchLater.length <= 0 ? <EmptyData /> :
+                        <div className='grid-30-70'>
+                            <PageInfoCard image_id={image_id} title="watchlater" />
+                            <div>
+                                <VerticalVideoCard
+                                    videos={watchLater}
+                                    // urlId={watchLaterId}
```
Comment (ravikumar0403) [inline src/pages/WatchLater/WatchLater.js:21]: remove commented code if not required

**Review by ravikumar0403 — APPROVED** [review]

**Review by dusariudaykumar — APPROVED** [review]
- Overall code is good
- Naming conventions are good
- The file structure is also good.

**Pull request merged**